src/Sarif.Driver/Sdk/PluginDriverCommand.cs

```diff
@@ -95,5 +178,42 @@ protected virtual void ProcessBaseline(IAnalysisContext context, T driverOptions
                 };
             }
         }
+
+        protected virtual void PostLogFile(IAnalysisContext context, T driverOptions, IFileSystem fileSystem)
```

Comment:
>IAnalysisContext context [](http://example.com/codeflow?start=43&length=24)

this parameter isn't used.

Reply:
just refactored to remove that.
thanks :)